src/index.js

```diff
@@ -586,36 +569,14 @@ walletRouter.post('/send-tx',
       isString: true,
       optional: true
     },
-    token: {
```

Comment:
I like this change, but we are changing the API of the `/send-tx`. It means all integrations using it must adapt their code. Should we create a new version of the API and mark this one as deprecated?

Reply:
Do you mean to create `/send-tx-new`? Or `/v2/send-tx`?

I agree that the integrations must adapt their code but it's only if they want to upgrade the headless.

I would need to map the old parameters to the new one and have two different APIs. I guess it's fair if we decide that it's important to do it.